packages/core/rollup.config.ts

```diff
@@ -0,0 +1,18 @@
+import { RollupOptions } from 'rollup';
+import { buildConfig } from '../../scripts/getRollupConfig';
+
+export const input = [
+  'src/index.ts',
+  'src/http/index.ts',
+  'src/rpc/index.ts',
+  'src/observable/index.ts',
+  'src/internals/index.ts',
+  'src/shared/index.ts',
+];
```

Comment:
This could probably be reduced so it's like fewer entrypoints in core

Reply:
yea as mentioned above i especially don't like the shared one